UPGRADING.rst

```diff
@@ -10,6 +10,23 @@ branch that does not have the listed changes, the steps would need to be
 reverted. This is all fairly informal and loosely defined. Hopefully we won't
 have too many entries in this file.
 
+#4011 Integrate monitoring SNS topic with Slack
+===============================================
+
+Operator
+~~~~~~~~
+
+Set up AWS Chatbot Slack client, `described
+<./README.md#313-aws-chatbot-integration-with-slack>`_ in our README. This has
+been done for the `platform-hca-dev` account and remains to be configured for
+`platform-hca-anvildev` and `platform-hca-prod`.
+
+Follow these steps to deploy for ``dev.shared``, ``anvildev.shared``, and
+``prod.shared``::
+
+    _select dev.shared
+    make -C $project_root/terraform/shared apply
```

Comment:
Take a look at the section right below this one on how to avoid the specifics. The operator should know how to deploy the `shared` component.